files/build_templates/dhcp_relay.service.j2

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=DHCP relay container
-Requires=updategraph.service swss.service teamd.service
+Requisite=updategraph.service swss.service teamd.service
```

Comment:
start of swss service will trigger start of dhcp_relay service which now will fail. because it is not guarantied that teamd will be already started as it is managed in the script in wait().

Checked this on DUT and got:
```
Nov 26 15:40:39.009962 arc-switch1025 INFO systemd[1]: Starting switch state service...
Nov 26 15:40:43.383200 arc-switch1025 INFO systemd[1]: Started switch state service.
Nov 26 15:40:43.383948 arc-switch1025 INFO systemd[1]: Starting SNMP container...
Nov 26 15:40:43.384887 arc-switch1025 NOTICE systemd[1]: TEAMD container is not active.
Nov 26 15:40:43.385738 arc-switch1025 WARNING systemd[1]: Dependency failed for DHCP relay container.
Nov 26 15:40:43.386313 arc-switch1025 NOTICE systemd[1]: dhcp_relay.service: Job dhcp_relay.service/start failed with result 'dependency'.
Nov 26 15:40:44.016929 arc-switch1025 INFO systemd[1]: Starting syncd service...
Nov 26 15:40:52.841680 arc-switch1025 INFO systemd[1]: Started SNMP container.
Nov 26 15:40:55.079201 arc-switch1025 INFO systemd[1]: Started syncd service.
Nov 26 15:40:55.099903 arc-switch1025 INFO systemd[1]: Starting TEAMD container...

```

Reply:
Thanks for your feedback, stepanblyschak. It appears that since the DHCP relay relies on both swss and teamd, and teamd is now managed in the wait() function of swss.sh, we may also need to add the dhcp_relay service to the swss.sh script and remove these dependencies altogether to ensure that the dhcp_relay service gets started in the appropriate order.